Loading data...
Process Process-1:
Traceback (most recent call last):
  File "/usr/common/software/pytorch/v1.2.0-gpu/lib/python3.6/multiprocessing/process.py", line 258, in _bootstrap
    self.run()
  File "/usr/common/software/pytorch/v1.2.0-gpu/lib/python3.6/multiprocessing/process.py", line 93, in run
    self._target(*self._args, **self._kwargs)
  File "/global/homes/d/danieltm/.local/cori/pytorchv1.2.0-gpu/lib/python3.6/site-packages/wandb/wandb_agent.py", line 63, in _start
    function()
  File "sweep.py", line 124, in train
    train_dataset, val_dataset = load_data(train_size=wandb.config.get("train_size",0), test_size=20)
  File "../utils/toy_utils.py", line 41, in load_data
    y_params=(torch.from_numpy(di[3][:,0]).unsqueeze(1)), pid=torch.from_numpy(di[4])) for di in train_graphs]
  File "../utils/toy_utils.py", line 41, in <listcomp>
    y_params=(torch.from_numpy(di[3][:,0]).unsqueeze(1)), pid=torch.from_numpy(di[4])) for di in train_graphs]
NameError: name 'torch_geometric' is not defined
